### ISSUE TITLE
Windows - error: failed to run custom build command for `tether v0.3.0`

### ISSUE THREAD

**Issue by asmarcz** [quadrupleslap/tether#23]
Hi, I am struggling with compiling on Windows. 

I use `stable-x86_64-pc-windows-msvc` toolchain and get this error:

```
D:\projects\rust\tether_test>cargo run
   Compiling tether v0.3.0
error: failed to run custom build command for `tether v0.3.0`
process didn't exit successfully: `D:\projects\rust\tether_test\target\debug\build\tether-314fb1adee4117e1\build-script-build` (exit code: 1)
--- stdout
cargo:rustc-link-lib=dylib=ole32
cargo:rustc-link-lib=dylib=user32
cargo:rustc-link-lib=dylib=windowsapp

--- stderr
Error: Os { code: 2, kind: NotFound, message: "Systém nemůže nalézt uvedený soubor." }
```
(the Error message says "System could not find the specified file.")

EDIT: I ran it in Native Tools Command Prompt from Visual Studio 2019

**Comment by quadrupleslap** [comment]
Thanks! Just checking — you have Make and Clang installed, right? It looks like the build script is complaining that it can't find Make. If you have Chocolatey, you can install it with:

```
choco install make
```

**Comment by asmarcz** [comment]
I didn't have Make in PATH. Sorry for bothering.

**Comment by quadrupleslap** [comment]
Okay, I'll try to improve the error message — "System could not find the specified file" is a lot more confusing than "we couldn't find make".

**Comment by quadrupleslap** [comment]
Closed by 16175f6.
